### ISSUE TITLE
Unable to reproduce results

### ISSUE THREAD

**Issue by ErikStammes** [zbf1991/RRM#7]
Hi,

Thanks for the great work. I'm trying to reproduce your results, but I can't seem to get close to the scores you report in the paper.

I have tried two settings:
- The default parameters as given in the code. This results in ~56.2 mIoU without CRF and ~57.6 mIoU with CRF
- With parameters as given in the paper (lower learning rate (0.001) and lower image resolution (321)). This results in ~55.4 without CRF and ~57.4 with CRF.

Other than those things, I have not made any changes. Any idea why my scores are much lower? Are there other parameters which I need to set?

Thanks!

**Comment by zbf1991** [comment]
I am not sure why your result is lower, do you train from init or from the pre-trained cls model? could you try again using train_cls_weight and the pre-trained classification model because it is more stable. Sorry for not solving your question.

**Comment by ErikStammes** [comment]
I used `train_from_init.py` but that already uses pretrained weights, right? i.e. `ilsvrc-cls_rna-a1_cls1000_ep-0001.params`. 

I'll try to use train_from_cls_weight now. What is the difference between `train_from_cls_weight.py` and `train_from_cls_weight(ori).py`?

**Comment by zbf1991** [comment]
Yeah. init used your mentioned model weights. 
no difference between these two file.  (ori) is just the older version. I will check init.py again. Maybe there is some mistake when I update it. Sorry for that.

**Comment by ErikStammes** [comment]
Alright, thanks for the clarification!

**Comment by ErikStammes** [comment]
I am now able to reproduce your results using `train_from_cls_weight.py`, thanks!

**Comment by wjun0830** [comment]
@zbf1991 Is the issue "train_from_init" solved?
I am trying to use another classification model and want to evaluate using your framework.
